### ISSUE TITLE
Match inadvertantly set into submitted state

### ISSUE THREAD

**Issue by Crossle86** [FIRST-Tech-Challenge/scorekeeper#172]
Our first match became corrupted, entering the submitted state. After that, the match remained  on the incomplete match list and we could not access the match and it was not reflected in the rankings. FIRSTWA had to take our db and manually correct the problem. We think the issue was clicking post before save but not sure. Again, once in submitted, we could replay the match but it just created a duplicate match on the incomplete match list and in no case could we update the match and get it into the correct state.

**Comment by adwiii** [comment]
Were you using live scoring? If not, was live scoring turned off in the Match Control Page settings? 

What was happening before the match got in the submitted state? Had the match been played, loaded for play, etc?

I have been able to get the match to display twice on the incomplete matches tab, but it appears that this is a display bug and would not have impeded your ability to commit the match. 

Also, I *think* that the Post button and the Commit/Save buttons should always be in opposite states - that is, you should not have the option to Save and Post at the same time. If this was an option, then that might be a clue as to what happened.

What browser were you using for the Match Control Page?

**Comment by Crossle86** [comment]
Live scoring off on all views. Chrome for all browsers. First match was played and second match loaded and started. While second match running, entered scores for first match. Now while score entry screen shown, the scorekeeper thinks she hit commit before save. The doc says commit will do save, so that suggests either button can be pressed and therefore would both be active. However, I did not see any of this. Scorekeeper called  me over after the match went into submitted  state. Its also possible the scorekeeper displayed some other view and then returned to score edit to finish and at that point the match was  in the wrong state. Since I did not see it I can't be sure of her steps, so I'm good with letting this one go if I am the only one reporting it.
